google/ads/google_ads/interceptors/interceptor.py

```diff
@@ -231,6 +231,10 @@ def _get_google_ads_failure(self, trailing_metadata):
             for kv in trailing_metadata:
                 if kv[0] == self._failure_key:
                     try:
+                        if not self._error_protos:
+                          self._error_protos = import_module(
```

Comment:
Do we want to wrap this in a try/except block? If the import fails we should fail elegantly with information on why we failed. Otherwise, I believe the next line will cause a stacktrace.

Reply:
Yes possibly - how do you think we should handle the failure case? Should this function return a different type of error? I also don't hate the idea of letting this raise a stack trace since a failure to load this module is not an expected error state.

Reply:
Does `import_module` raise an exception on failure? I'd certainly hope so.

Reply:
I'm leaning towards keeping this as-is. `import_module` does raise on failure, and we wouldn't really be able to provide a much better custom message than "could not load this error module."

Reply:
Rethrowing is the right thing to do, but I'm not sure how to surface the "could not load this error module". Can an error logging suffice? I mean, if we rethrow, the users will get an RpcException whereas they anticipate a GoogleAdsException, right? It will lead to failure of another form immediately after?

Reply:
They will get a [ModuleNotFoundError](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError), which is more correct than rethrowing GoogleAdsException

Reply:
I think we should **not** wrap and return a friendly message. Otherwise the customer will have trouble diagnosing the root cause. It also sets a bad precedent for catching any number of system errors, which is a bad practice.